src/platform/web/Platform.js

```diff
@@ -178,6 +180,9 @@ export class Platform {
             this._serviceWorkerHandler,
             this._config.push
         );
+        const manifests = this.config["themeManifests"];
+        await this._themeLoader?.init(manifests);
+        await this._themeLoader?.loadThemeFromSetting();
```

Comment:
Why not just loadThemeFromSettings(await themeLoader.getActiveTheme())? That will also apply the default theme if you don't have a preference set yet.

Reply:
did you mean `themeLoader.setTheme(await themeLoader.getActiveTheme())`?